solidity/contracts/converter/types/liquidity-pool-v2/LiquidityPoolV2Converter.sol

```diff
@@ -156,6 +167,17 @@ contract LiquidityPoolV2Converter is LiquidityPoolConverter {
         emit Activation(converterType(), anchor, true);
     }
 
+    /**
+      * @dev updates the current dynamic fee factor
+      * can only be called by the contract owner
+      *
+      * @param _dynamicFeeFactor new dynamic fee factor, represented in ppm
+    */
+    function setDynamicFeeFactor(uint256 _dynamicFeeFactor) public ownerOnly {

```

Comment:
limit the factor to 10%, should be enough and will prevent abuse

Reply:
Worse, we must verify that `factor <= 100% / AMP` (which means 5%), otherwise the dynamic fee might be larger than 100%.